### PR TITLE
Add Master - Chance of Admit

### PR DESCRIPTION
**Pull request by BBloggsbott**
Added a new project: [Masters - Chance of Admit](https://masters-chance-of-admit.herokuapp.com/)

### CODE REVIEW

File: _data/projects/masters-chance-of-admit.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/BBloggsbott/masters-chance-of-admit
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- html
+- machine-learning
+- data-viz
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/BBloggsbott/masters-chance-of-admit/issues
```
Comment (ritwik12) [inline _data/projects/masters-chance-of-admit.yml]: @BBloggsbott Please change this to https://github.com/BBloggsbott/masters-chance-of-admit/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/masters-chance-of-admit.yml (hunk 1 above)
Comment (BBloggsbott) [reply _data/projects/masters-chance-of-admit.yml]: The requested changes have been made

Comment (ritwik12) [reply _data/projects/masters-chance-of-admit.yml]: Sorry I gave the wrong URL, your tag is "up-for-grabs" so please change the link to https://github.com/BBloggsbott/masters-chance-of-admit/labels/up-for-grabs

Comment (ritwik12) [reply _data/projects/masters-chance-of-admit.yml]: @BBloggsbott ^^

Comment (BBloggsbott) [reply _data/projects/masters-chance-of-admit.yml]: Done

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/masters-chance-of-admit.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/masters-chance-of-admit.yml]: Gotcha

**Comment by ritwik12** [comment]
@BBloggsbott Thanks!!

**Pull request merged**